src/main/java/com/example/tickets/security/SecurityConfig.java

```diff
@@ -65,19 +105,25 @@ protected void configure(HttpSecurity http) throws Exception {
                 .sessionManagement().sessionCreationPolicy(SessionCreationPolicy.STATELESS)
                 .and()
                 .authorizeRequests()
+                .antMatchers("/api/u/**").authenticated()
                 .antMatchers(LOGIN_ENDPOINT).permitAll()
```

Comment:
.permitAll() should be replaced by .anonymous() - login action should be available only to unauthenticated users